File: src/main/java/jkmau5/alternativeenergy/world/item/AltEngItems.java
package jkmau5.alternativeenergy.world.item;

import cpw.mods.fml.common.registry.GameRegistry;

public class AltEngItems {

    public static ItemUpgrade itemUpgrade;
    public static ItemAlternativeWrench itemAlternativeWrench;
    public static ItemWrenchUpgrade itemWrenchUpgrade;

    public static void init() {
        itemUpgrade = new ItemUpgrade();
        itemAlternativeWrench = new ItemAlternativeWrench();
        itemWrenchUpgrade = new ItemWrenchUpgrade();
        //GregTech_API.registerWrench(new ItemStack(itemAlternativeWrench));
        //GregTech_API.sWrenchList.add(Integer.valueOf(GT_Utility.stackToInt(new ItemStack(itemAlternativeWrench))));

        //The names in here are the new itemIDs. Do not change them!
        GameRegistry.registerItem(itemUpgrade, "upgrade");
        GameRegistry.registerItem(itemAlternativeWrench, "alternativeWrench");
        GameRegistry.registerItem(itemWrenchUpgrade, "wrenchUpgrade");
    }
}

File: src/main/scala/alternativemods/alteng/items/AltEngItems.scala
package alternativemods.alteng.items

import cpw.mods.fml.common.registry.GameRegistry

object AltEngItems {

  var itemEnergyConsumer: ItemEnergyConsumer = _
  def load(){
    itemEnergyConsumer = new ItemEnergyConsumer()

    GameRegistry.registerItem(itemEnergyConsumer, "itemEnergyConsumer")
  }

}
